packages/blade/scripts/generateCssThemeVariables.js

```diff
@@ -4,6 +4,7 @@ const set = require('lodash/set');
 const cloneDeep = require('lodash/cloneDeep');
 
 const { paymentTheme, bankingTheme } = require('../build/js-bundle-for-css/tokensBundle');
+// Todo : confirm if any change is needed for these imports
```

Comment:
is it breaking somewhere?

Reply:
Ohhhh understood what the issue is. Things we don't export from `index.ts` don't end up in `utilsBundle` 

And we have this here - 

```ts
const {
  makeBorderSize,
  makeMotionTime,
  makeSpace,
  makeTypographySize,
  makeSize,
} = require('../build/js-bundle-for-css/utilsBundle');
```

where we are importing utilities which are now internal utilities.

To get unblocked, maybe we can make these utilities as external utilities and re-export from `index.ts` of utils. That should make builds working

Reply:
Can later figure out solution to this.

Reply:
1. yes, `generateCssThemeVariables.js` is trying to import the following utilities which were earlier exported from index file.
```js
const {
  makeBorderSize,
  makeMotionTime,
  makeSpace,
  makeTypographySize,
  makeSize,
} = require('../build/js-bundle-for-css/utilsBundle');
```
2. `build:generate-css-variables` is failing because of this.

Reply:
lets mark them external for now. We'll figure out solution later. // cc @divyanshu013